src/test/app/DepositAuth_test.cpp

```diff
@@ -724,14 +792,506 @@ struct DepositPreauth_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testCredentials()
+    {
+        using namespace jtx;
+
+        const char credType[] = "abcde";
+        Account const issuer{"issuer"};
+        Account const alice{"alice"};
+        Account const bob{"bob"};
+        Account const john("john");
```

Comment:
`john` doesn't appear to be used at all.

Reply:
fixed